src/ParallelAlgorithms/Events/ObserveTimeStep.hpp

```diff
@@ -101,13 +130,20 @@ class ObserveTimeStep : public Event<EventRegistrars> {
         "without a preceding '/'."};
   };
 
+  struct OutputTime {
+    using type = bool;
+    static constexpr Options::String help = {
+        "Whether to print the time to screen "
+        "using the FormatTimeOutput formatter."};
```

Comment:
delete the last part, the person writing the input file doesn't care about implementation details :)